data/human/hails.txt

```diff
@@ -2708,6 +2708,196 @@ phrase "friendly civilian"
+		"Farseer"
+		"Pilot"
+		"Ada"
+		"Hermes"
+		"Mainsail"
+	word
+		" "
+	word
+		"blackjack."
+		"poker."
+		"backgammon."
+		"baccarat."
+		"roulette."
+
+phrase "friendly civilian"
+	word
+		"Have you read the latest edition of"
+	word
+		" "
+	word
+		"Cosmic Chronicles"
+		"Daring Discoveries"
+		"Norn Nebula News"
+		"Southern Rim Spacer News"
+		"Stars and Starships"
+		"Sunracer Stellar Spectacles"
+		"Tarazed Today"
+		"The Galactic Gazette"
+		"The New Moonshake"
+		"The Satellite Sentinel"
+	word
+		"? "
+	word
+		"It says that"
+		"It claims that"
+		"It says"
+		"It claims"
+	word
+		" "
+	word
+		"pulse showers are good for your quantum pores"
+		"exposing yourself to neodymium increases your risk of spacepox"
+		"shoddily grown ion algae exhausts your psychic karma"
+		"laser acupuncture lets you communicate with nebula ghosts"
+		"star crystals bring luck to long distance relationships"
+		"consuming moon dust accelerates hair growth but causes lunacy"
+		"excessive exposure to zero gravity can lead to phantom limb syndrome"
+		"alien prosthetics often come with unexpected telepathic abilities"
+		"stardate tattoos increase your chances of alien abduction"
+	word
+		"."
+		"!"
+
+phrase "friendly civilian"
+	word
+		"I'm no weapons expert, but it feels better firing"
+	word
+		" "
+	word
+		"energy blasters"
+		"particle cannons"
+		"proton guns"
+	word
+		" "
+	word
+		"than"
+	word
+		" "
+	word
+		"beam lasers."
+		"heavy lasers."
+
+phrase "friendly civilian"
+	word
+		"I shouldn't have sold my old ship. It was just a"
+	word
+		" "
+	word
+		"Shuttle"
+		"Star Barge"
+		"Sparrow"
+		"Heavy Shuttle"
+		"Freighter"
+		"Fury"
+	word
+		" "
+	word
+		"but it never had"
+	word
+		" "
+	word
+		"stalling engines."
+		"unstable shielding."
+		"overheating fuel cells."
+		"a glitchy HUD."
+		"faulty ship detection."
+
+phrase "friendly civilian"
+	word
+		"Finished cleaning my cargo hold. The"
+		"Cargo hold is all spick and span. The"
+		"Finished scrubbing the hold. The"
+	word
+		" "
+	word
+		"freshener"
+		"perfume"
+		"spray"
+		"fragrance"
+	word
+		" "
+	word
+		"I used makes it smell like"
+	word
+		" "
+	word
+		"strawberries"
+		"bananas"
+		"peaches"
+		"roses"
+		"jasmine"
+		"lavender"
+	word
+		"."
+		"!"
+
+phrase "friendly civilian"
+	word
+		"Have you seen the holofilm where the"
+	word
+		" "
+	word
+		"good guy"
+		"hero"
+		"bad guy"
+		"villain"
+	word
+		" "
+	word
+		"mounts overcharged twin blasters on a modded Argosy"
+		"shoots up a security station with dual laser pistols"
+		"dodges quad blaster turrets in a stolen Flivver"
+	word
+		"? So cool."
+
+phrase "friendly civilian"
+	word
+		"Frankly, I'm thinking of joining the Navy because"
+		"Honestly, I'm going to join the Navy because"
+		"I'm enlisting in the Navy because"
+	word
+		" "
+	word
+		"of their dental insurance."
+		"they'll cover my tuition."
+		"to see if I can get my hands on one of their warships."
+		"they offer childcare assistance."
```

Comment:
Whoops, doing it this way gives a chance of a "because to see" combination.
```suggestion
	word
		"Frankly, I'm thinking of joining the Navy"
		"Honestly, I'm going to join the Navy"
		"I'm enlisting in the Navy"
	word
		" "
	word
		"because of their dental insurance."
		"since they'll cover my tuition."
		"to see if I can get my hands on one of their warships."
		"because they offer childcare assistance."
```